src/config_utils.rs

```diff
@@ -22,10 +23,68 @@ use std::path::PathBuf;
 use std::env;
 use rustc_serialize::json;
 use std::io;
+use rustc_serialize::{Decodable, Decoder, Encodable, Encoder};
+
+#[derive(PartialEq, Eq, Hash, Debug, Clone)]
```

Comment:
This code is failing to compile now, with latest rust nightly:

```
   Compiling crust v0.1.5 (file:///home/vinipsmaker/Projetos/maidsafe/crust)
src/config_utils.rs:30:5: 30:28 error: the trait `core::hash::Hash` is not implemented for the type `time::Tm` [E0277]
src/config_utils.rs:30     pub timestamp: time::Tm,
                           ^~~~~~~~~~~~~~~~~~~~~~~
src/config_utils.rs:28:25: 28:29 note: in expansion of #[derive_Hash]
src/config_utils.rs:28:25: 28:29 note: expansion site
src/config_utils.rs:30:5: 30:28 help: run `rustc --explain E0277` to see a detailed explanation
error: aborting due to previous error
Could not compile `crust`.
```

I can update it, but I'd like to confirm it was working before, so I may investigate how to mimic previous behaviour.